pkg/exporter/sumologicexporter/sender_test.go

```diff
@@ -765,6 +819,71 @@ func TestOverrideSourceHost(t *testing.T) {
 	})
 }
 
+func TestLogsDontSendSourceFieldsInXSumoFieldsHeaderWhenCorrespondingXSumoHeaderIsSet(t *testing.T) {
```

Comment:
not only for `CorrespondingXSumoHeaderIsSet`

Reply:
👍